tests/longstitch_tests.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for ntJoin"""
```

Comment:
Did you mean `LongStitch` here? :smile:

Reply:
Oh yes good catch! :D I'm busted for my copy/pasting from the ntJoin pytests haha

Reply:
Hahaha we all do it!!